get_azure_unique_id

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#
+# get_azure_unique_id: prints a unique id for an Azure VM suitable for use with SignalFx.
+# exits with 0 if the unique id was generated succesfully
+# exits with 1 if the curl for data succeeded but the parsing of the information failed.
+# exits with 28 if there was a timeout talking to Azure to get information
+# all other exit codes come from curl
+#
+
+jsonval () {
+    temp=`echo $json | sed 's/\\\\\//\//g' | sed 's/[{}]//g' | awk -v k="text" '{n=split($0,a,","); for (i=1; i<=n; i++) print a[i]}' | sed 's/\"\:\
```

Comment:
Wow this is some amazing sed magic but is there no easier way to get this data from the API?

Reply:
The API gives out a nested json object with all metadata about the VM and we need to parse through it get these pieces of information